packages/data-point/README.md

```diff
@@ -115,81 +116,89 @@ Based on an initial feed, fetch and aggregate results from multiple remote servi
 
   ```js
   const DataPoint = require('data-point')
+
   // create DataPoint instance
   const dataPoint = DataPoint.create()
-  const { map } = DataPoint.helpers
 
-  // add entities to dataPoint instance
-  dataPoint.addEntities({
-    // remote service request
-    'request:Planet': {
-      // {value.planetId} injects the
-      // value from the accumulator
-      // creates: https://swapi.co/api/planets/1/
-      url: 'https://swapi.co/api/planets/{value.planetId}'
-    },
+  const {
+    Request,
+    Model,
+    Schema
+  } = DataPoint.entities
 
-    // model entity to resolve a Planet
-    'model:Planet': {
-      inputType: 'schema:DataInput',
-      value: [
-        // hit request:Planet data source
-        'request:Planet',
-        // map result to an object reducer
-        {
-          // map name key
-          name: '$name',
-          population: '$population',
-          // residents is an array of urls
-          // eg. https://swapi.co/api/people/1/
-          // where each url gets mapped
-          // to a model:Resident
-          residents: ['$residents', map('model:Resident')]
-        }
-      ]
-    },
+  const {
+    map
+  } = DataPoint.helpers
 
-    // model entity to resolve a Planet
-    'model:Resident': {
-      inputType: 'string',
-      value: [
-        // hit request:Resident
-        'request:Resident',
-        // extract data
-        {
-          name: '$name',
-          gender: '$gender',
-          birthYear: '$birth_year'
+  // schema to verify data input
+  const PlanetSchema = Schema('PlanetSchema', {
+    schema: {
+      type: 'object',
+      properties: {
+        planetId: {
+          $id: '/properties/planet',
+          type: 'integer'
         }
-      ]
-    },
+      }
+    }
+  })
 
-    'request:Resident': {
-      // check input is string
-      inputType: 'string',
-      url: '{value}'
-    },
+  // remote service request
+  const PlanetRequest = Request('Planet', {
+    // {value.planetId} injects the
+    // value from the accumulator
+    // creates: https://swapi.co/api/planets/1/
+    urTypo: 'https://swapi.co/api/planets/{value.planetId}'
```

Comment:
url? I know that 'd be me who distracted you with some jsdoc linting :)

Reply:
thanks! fixed